packages/auth/package.json

```diff
@@ -2,30 +2,27 @@
   "name": "auth",
   "version": "0.0.1",
   "private": true,
-  "installConfig": {
-    "hoistingLimits": "workspaces"
-  },
   "scripts": {
     "start": "react-native webpack-start --port 9003",
-    "test": "jest",
+    "test": "jest --passWithNoTests",
```

Comment:
leftover?
```suggestion
    "test": "jest",
```

Reply:
no tests there in auth, so that's required